BeyondCorpClientGateways/composer.json

```
@@ -0,0 +1,35 @@
+{
+    "name": "google/cloud-beyondcorp-clientgateways",
+    "description": "Google Cloud BeyondCorp Client Gateways for PHP Client for PHP",
+    "license": "Apache-2.0",
+    "minimum-stability": "stable",
+    "autoload": {
+        "psr-4": {
+            "Google\\Cloud\\BeyondCorp\\ClientGateways\\": "src",
+            "GPBMetadata\\Google\\Cloud\\Beyondcorp\\Clientgateways\\": "metadata"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Google\\Cloud\\BeyondCorp\\ClientGateways\\Tests\\": "tests"
+        }
+    },
+    "extra": {
+        "component": {
+            "id": "cloud-beyondcorp-clientgateways",
+            "path": "BeyondCorpClientGateways",
+            "entry": null,
+            "target": "googleapis/google-cloud-php-beyondcorp-clientgateways.git"
+        }
+    },
+    "require": {
+        "google/gax": "^1.17.0"
+    },
+    "require-dev": {
+        "phpunit/phpunit": "^4.8|^5.0|^8.0"
```

nit, but we can drop PHP 4.8, since 5.0 is compatible with PHP 5.6
```suggestion
        "phpunit/phpunit": "^5.0|^8.0"
```